assets/javascripts/kitten/components/form/form-amount-and-currency.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import SelectWithState from 'kitten/components/form/select-with-state'
+
+class FormAmountAndCurrency extends React.Component {
```

Comment:
Note pour plus tard : j'avais testé les composants dits "fonctionnels" (https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components) et la compilation n'avait pas marché. J'aimerais retester cela pour tous les composants qui ne font que du `render`, ça allègerait un peu le code.